lib/libutils/isoc/include/setjmp.h

```diff
@@ -38,8 +38,10 @@
 /*
  * All callee preserved registers:
  * v1 - v7, fp, ip, sp, lr, f4, f5, f6, f7
+ * One additional reserved register used in case
+ * ftrace in enabled to restore ftrace return stack.
  */
-#define _JBLEN 23
+#define _JBLEN 24
```

Comment:
Earlier I also had this confusion. But after exploration, the float registers came out to be 96 bits in length (that makes the additional 8 count). So that makes the count correct.

Reply:
OK, thanks. I suggest `s/reserved register/32-bit value` because it's not really a register that you're saving at the time of this call.

Reply:
ok